docs/manifest.json

```diff
@@ -175,6 +175,11 @@
           "description": "Use docker inside containerized templates",
           "path": "./templates/docker-in-workspaces.md",
           "icon_path": "./images/icons/docker.svg"
+        },
+        {
+          "title": "Inheritance",
```

Comment:
I suggest calling this `Code Reuse` or `Sharing Code` instead of Inheritance. It strikes closer to the problem that the user is having.

Reply:
@ammario @bpmct why don't we directly reference the Terraform terminology and name it `Terraform Modules` ?

Reply:
@ericpaulsen sounds good to me too